Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -80,10 +80,10 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
       }
 
     self.goToRewards = goToRewards
-      .filter { _ in featureNativeCheckoutPledgeViewEnabled() }
```

Comment:
Make sure that when merging `master` this is actually using the correct feature flag `Native Checkout` not `Native Checkout Pledge View` ... @justinswart has recently changed it since it was causing issue with deep links 😅

Reply:
Yeah, I noticed that. Actually we now will use  `userCanSeeNativeCheckout` because of the experiment 😄